src/main/java/com/growup/todoary/exception/NotExistentTodoException.java

```diff
@@ -0,0 +1,8 @@
+package com.growup.todoary.exception;
+
+public class NotExistentTodoException extends RuntimeException{
```

Comment:
네이밍이 약간 읽기 어렵다는 느낌을 받았어요
존재하지 않는 Todo... IllegalStateException과 같은 느낌이 드는데 자원의 관점에서 생각하려다보니 위와같은 네이밍이 도출된 것 같아요
자원에 접근하려는 클라이언트(개발자)의 관점에서 네이밍을 생각해보는건 어떤가요?
예를들면 다음과 같은 네이밍이 떠오르네요
`TodoNotFoundException`

+) 사소한 부분이지만 공백이 하나 없네요
```suggestion
public class NotExistentTodoException extends RuntimeException {
```

Reply:
`TodoNotFoundException`이 훨씬 나아 보이네요..!